.github/workflows/releases.yml

```diff
@@ -282,78 +282,6 @@ jobs:
             REPO_URL=sumologic/sumologic-otel-collector \
             BUILD_TYPE_SUFFIX="-ubi"
 
-  # Store the install script from the packaging repository as a release artifact.
```

Comment:
So to confirm, all the install scripts only live in the packaging repository as of the latest release workflow?

Reply:
They still exist within this repository but they do not get included with the release. The Sumo website is also using the install scripts in the packaging repo since the release workflow changes went into effect.